src/commands/config-get.json

```diff
@@ -0,0 +1,24 @@
+{
+    "GET": {
```

Comment:
Note the following doc pr: https://github.com/redis/redis-doc/pull/1705
And sister code pr: #9914
Once merged this should be updated.